libraries/rush-lib/src/logic/operations/IOperationExecutionResult.ts

```diff
@@ -43,6 +43,11 @@ export interface IOperationExecutionResult {
    * The id of the runner which actually runs the building process in cobuild mode.
    */
   readonly cobuildRunnerId: string | undefined;
+
+  /**
+   * Returns true if this operation was co-built using this machine, false if cobuilds are disabled or it was executed on another agent.
```

Comment:
It seems like it'd make more sense for this to be `true` if cobuilds are disabled, since operations will always be executed on the current machine in that case.

Reply:
Good point, I'll update

Reply:
Updated in [02de4c8](https://github.com/microsoft/rushstack/pull/4680/commits/02de4c881b136cdd06aee539b1b41f95ceeb3e57)